app/routes/overview/components/Overview.css

```diff
@@ -31,6 +31,23 @@
   width: 100%;
 }
 
+.header {
+  padding-top: 20px;
+  padding-left: 10px;
+  padding-right: 10px;
+  flex: 1;
+  width: 100%;
+  color: #666;
+  text-transform: uppercase;
+  font-weight: 400;
+  letter-spacing: 1px;
+
+  @media (--mobile-device) {
+    padding: 0;
```

Comment:
If you give the media version of the header 0 padding if will float all the way to left.
![screenshot from 2018-03-22 16-37-36](https://user-images.githubusercontent.com/23152018/37780633-6e50eaf4-2def-11e8-8f85-84500f6c9090.png)
The other headings on the front page get left paddings from its parent elements
![screenshot from 2018-03-22 16-37-55](https://user-images.githubusercontent.com/23152018/37780702-90d36872-2def-11e8-9063-505f68a0ff54.png)

I think setting `padding: 0 0 0 10px` will have the intended effect.

It would also be nice if `Andre oppslag` had the same spacing down to its content as `Festet oppslag`
![screenshot from 2018-03-22 16-46-04](https://user-images.githubusercontent.com/23152018/37781475-101b8924-2df1-11e8-9bb0-88d61919da47.png)
 The front page events under `Andre oppslag` have a preset padding-top, and the `h2` header gets a padding, and with the addition of the `margin-bottom: 10px` it total's up to a lot spacing.